docs/fundamentals/dashboard/standalone-for-python.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Using Aspire Dashboard with Python Applications
+description: How to use the Aspire Dashboard in a Python application.
+ms.date: 08/27/2024
+ms.topic: tutorial
+ms.author: aapowell
+---
+
+# Tutorial: Using the Aspire Dashboard with Python Applications
+
+The [Aspire dashboard](overview.md) provides a great UI for viewing telemetry, and is available as a standalone container image that can be used with any OpenTelemetry-enabled app. In this article, you'll learn how to:
+
+> [!div class="checklist"]
+>
+> - Start the Aspire dashboard in standalone mode.
+> - Use the Aspire dashboard with a Python application.
+
+## Prerequisites
+
+To complete this tutorial, you need the following:
+
+- [Docker](https://docs.docker.com/get-docker/)
+  - You can use an alternative container runtime, but the commands in this article are for Docker.
+- [Python 3.9 or higher](https://www.python.org/downloads/) locally installed
+- A sample application
+
+## Sample application
+
+This quickstart can be completed using either Flask, Django, or FastAPI. A sample application in each framework is provided to help you follow along with this quickstart. Download or clone the sample application to your local workstation.
```

Comment:
Now we're a quickstart? We're either a tutorial or a quickstart, not both 😄. Since we have prerequisites listed, that usually implies a tutorial.

```suggestion
This tutorial can be completed using either Flask, Django, or FastAPI. A sample application in each framework is provided to help you follow along with this tutorial. Download or clone the sample application to your local workstation.
```